zulip_bots/zulip_bots/lib.py

```diff
@@ -156,6 +158,12 @@ def run_message_handler_for_bot(lib_module, quiet, config_file, bot_name):
 
     state_handler = StateHandler()
 
+    # Look in bot class to see if empty messages will be handled in this function
+    try:
+        handle_empty_here = not (lib_module.handler_class.ACCEPT_EMPTY_MESSAGES)
+    except AttributeError:
+        handle_empty_here = True
```

Comment:
Nice catch of a pattern, but if this is changed, I think it probably should be
```
handle_empty = not getattr(lib_module.handler_class, 'ACCEPT_EMPTY_MESSAGES', False)
```
That is, if it gets the attr, then True there (bot handles it) implies False to `handle_empty` (lib handles it); if not present then the result is the same as with yours.

RIght?